src/components/postFooter/GifContainer.jsx

```diff
@@ -114,11 +118,13 @@ const GifContainer = ({ displayTenor, removeTenor, handleSelectedGif }) => {
       </li>
     );
   });
+
   // Run the category search stuff once the page loads
   useEffect(() => {
     getCategories();
     // eslint-disable-next-line
```

Comment:
why are we disabling next line?

Reply:
I was getting an eslint error about my dependency array being empty or needing a particular variable that I didn't think was necessary so I disabled it